envs/example/defaults.yml

```diff
@@ -21,6 +21,7 @@ secrets:
   horizon_secret_key:           asdf
   glance_sync:      ADQ64XUQLUWH75M634RVBLP55RKPGGOWG
   monitor_password: asdf
+  telemetry_password: asdf
```

Comment:
If we keep them separate, it would make sense to name the variable in a way that indicates it's for rabbit.